Automattic/jetpack
# My Jetpack: Register `#/add-videopress` route

This PR registers the `#/add-videopress` route.
Additionally, it compresses the VideoPress interstitial image.

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: Register `#/add-videopress` route

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Open a tab with the `#/add-videopress` route
`https://your-testing-site/wp-admin/admin.php?page=my-jetpack#/add-videopress`
* Confirm you see the VideoPress interstitial page
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/77539/153939805-b61c7b8d-784d-4b35-8e68-99490b956966.png">
